modules/ROOT/pages/scopes-and-collections.adoc

```diff
@@ -0,0 +1,50 @@
+
+==== Cloud-to-Edge support for Scopes and Collections
+
+A Sync Gateway database is associated with a single Scope and can be related to one or more Collections within the Scope, including all collections within the Scope. The most significant benefit of this feature is enhanced data organization and management, leading to improved scalability, security, flexibility, and simplified development.
+
+==== Sync Gateway Metadata Isolation
+
+Scopes and Collections isolate Sync Gateway metadata. System data used by Sync Gateway is maintained in _default scope/collection, where it is logically isolated from application data in user-defined scopes/collections. The most crucial benefit of this feature is the separation of system data from application data, which results in improved data organization, security, and maintainability.
+
+==== Inter-Cluster Synchronization Gateway Replication
+This feature enables improved syncing of one or more data scopes across Sync Gateway clusters by employing a resilient replication protocol tailored explicitly for inter-Gateway communication. This robust Inter-Gateway replication protocol ensures seamless data synchronization, enhancing system reliability and performance.
+
+== Sync Gateway’s Function
+
+Sync Gateway's primary function in Scopes and Collections is facilitating data synchronization between Couchbase Lite clients and Couchbase Server. It does so by handling the following tasks:
+
+==== Segregation of Data
+
+Sync Gateway allows you to segregate data into different scopes within a single bucket and logically group documents of specific types into their collections. This improves data organization and simplifies access control policies.
+
+==== Simplified Access Control
+
+Sync Gateway simplifies access control and replication configurations by setting sync functions at the collection level. App users can be segregated by scope, ensuring they only access the authorized data.
+
+==== Scalable Multi-tenant Applications
+
+Sync Gateway supports multi-tenant applications that do not need strict resource isolation but still require independent data lifecycle management. Each tenant can have its own scope, allowing for easy addition or removal of tenants without impacting others.
+
+==== High Application Density at the Edge 
+
+Sync Gateway enables large-scale cloud-to-edge deployments with numerous applications at the edge, each served using a microservices-based application architecture. Scopes and collections allow for bucket consolidation, making the solution more scalable and cost-effective.
+
+==== Sync Gateway Metadata Isolation
+
+Sync Gateway can isolate system metadata from application data by storing it in the _default scope/collection. This separation ensures that SDK-based applications do not inadvertently modify sync metadata and simplifies query implementations.
```

Comment:
Either `_default` in backticks to display like code, or just default without the underscore maybe?